test/functional/fixtures/regression/gh-7482/test.js

```diff
@@ -0,0 +1,5 @@
+describe('[Regression](GH-7482)', () => {
+    it('Shouldn\'t throw errors if test and fixtures methods are executed before test and fixtures', async () => {
```

Comment:
It's better do not escape the `'` in the test name.
If this test will fail, you can't find it using Ctrl+C, Ctrl+F in IDE.
Surround the string in double quotes.
```js
it("Should n't throw errors if test and fixtures methods are executed before test and fixtures",
```